core/src/main/scala/vegas/DSL/EncoderDSL.scala

```diff
@@ -40,10 +40,11 @@ trait EncoderDSL[T] extends BaseEncoderDSL[T] {
     (field: OptArg[String] = NoArg, dataType: OptArg[Type] = NoArg,
      value: OptArg[Any] = NoArg,
      aggregate: OptArg[AggregateOp] = NoArg, axis: OptArg[Axis] = NoArg, hideAxis: OptArg[Boolean] = NoArg,
-     scale: OptArg[Scale] = NoArg, timeUnit: OptArg[TimeUnit] = NoArg, title: OptArg[String] = NoArg) = {
+     scale: OptArg[Scale] = NoArg, timeUnit: OptArg[TimeUnit] = NoArg, title: OptArg[String] = NoArg,
+     bin: OptArg[Bin] = NoArg, disableBin: OptArg[Boolean] = NoArg) = {
```

Comment:
Maybe enableBin instead? Since by default binning is disabled, and you'd expect the argument to switch it on, rather than off. That was my logic behind hideAxis too. That by default the axis is always On. So specifying an argument means changing the default situation.

What does everyone else find more initiative? @datamusing @dbtsai